### PR TITLE
added deprecation warnings and label names

### PR DESCRIPTION
**Pull request by rafaelhbarros**
related to anaconda-server/anaconda-server#1313

### CODE REVIEW

File: binstar_client/commands/copy.py [hunk 1]
```diff
@@ -13,10 +13,20 @@ def main(args):
 
     spec = args.spec
     channels = bs.list_channels(spec.user)
-    if args.from_channel not in channels:
-        raise errors.UserError("Channel %s does not exist\n\tplease choose from: %s" % (args.from_channel, ', '.join(channels)))
+    __name = 'label' if (args.from_label and args.to_label) else 'channel'
```
Comment (dsludwig) [inline binstar_client/commands/copy.py]: why are you using double-underscore prefix?

Comment (rafaelhbarros) [inline binstar_client/commands/copy.py]: something internal that has a simple name, I don't want to confuse this with a parameter or something. I can add comment if you think it's not clear.

Comment (dsludwig) [inline binstar_client/commands/copy.py]: I guess my point is that it's easy to confuse this with `__name__`. How about `label_text`? Or maybe just move the expression inline into the format statement:

```
"{label} {from_label} does not exist".format(
  label='Label' if (args.from_label and args.to_label) else 'Channel',
  from_label=from_label,
)
```

**Comment by dsludwig** [comment]
LGTM

**Pull request merged**